edm4hep.yaml

```diff
@@ -401,7 +401,7 @@ datatypes:
       - float                phi [rad]       // Azimuthal angle of the cluster's intrinsic direction (used e.g. for vertexing). Not to be confused with the cluster position seen from IP
       - edm4hep::Vector3f    directionError [mm^2]  // covariance matrix of the direction
     VectorMembers:
-      - float   shapeParameters      // shape parameters. This should be accompanied by a descriptive list of names in the shapeParameterNames collection level metadata, as a vector of strings with the same ordering
+      - float   shapeParameters      // shape parameters. The corresponding names of the shape parameters should be stored in the collection-level metadata collection identified by edm4hep::labels::ShapeParameterNames, as a vector of strings in the same order as the parameters.
```

Comment:
Technically we don't really have collection-level metadata any longer. We have a dedicated naming convention for file-level metadata for "attaching" some parameters to a given collection. However, I don't know if that can be conveyed easily in some way.